website/public/demo/error-message.mjs

```diff
@@ -0,0 +1,57 @@
+function showErrorMessageBox(mark) {
+  const div = createErrorBox(
+    mark,
+    mark.attributes["data-message"].value,
+    mark.attributes["data-code"].value,
+    mark.attributes["data-severity"].value
+  );
+  let body = document.querySelector("body");
+  body.appendChild(div);
+}
+
+function createErrorBox(markedElement, errorMessage, code, severity) {
+  // TODO: Change background color based of the severity
+  let div = document.createElement("div");
+  const { bottom, left, height } = markedElement.getBoundingClientRect();
+  div.setAttribute("id", "error-message-box");
+  div.innerText = `${code} - ${errorMessage}`;
+  div.setAttribute(
+    "style",
+    `position: fixed; 
+    top: ${Math.trunc(bottom)}px; 
+    left: ${Math.trunc(left)}px`
+  );
+  div.classList.add("error-box");
+  return div;
+}
+
+function removeErrorMessageBox() {
+  const errorMessagesBoxs = document.querySelectorAll("#error-message-box");
+  errorMessagesBoxs.forEach((box) => box.remove());
```

Comment:
If there can be multiple elements with ID error-message-box, then we should use a class or a JS array instead of an ID.

If there can only be zero or one elements with ID error-message-box, then I think it's a bit confusing to loop here.